testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DbUtils.kt

```diff
@@ -85,12 +87,15 @@ object DbUtils {
 
         val user = dbUser ?: getAdminUser()
         val password = dbPassword ?: getAdminPassword()
-        if(!schemaName.isNullOrBlank()) {
+        if (!schemaName.isNullOrBlank()) {
             if (createSchema) {
                 logger.info("Creating schema: $schemaName".emphasise())
                 factory.create(jdbcUrl, user, password, maximumPoolSize = 1).connection.createSchema(schemaName)
             }
-            jdbcUrl = "$jdbcUrl?currentSchema=$schemaName&reWriteBatchedInserts=true"
+            jdbcUrl = if (rewriteBatchedInserts)
+                {"$jdbcUrl?currentSchema=$schemaName&reWriteBatchedInserts=true"}
+                else
+                {"$jdbcUrl?currentSchema=$schemaName"}
```

Comment:
```suggestion
            jdbcUrl = if (rewriteBatchedInserts) {
                "$jdbcUrl?currentSchema=$schemaName&reWriteBatchedInserts=true"
            } else {
                "$jdbcUrl?currentSchema=$schemaName"
            }
```

I believe this is the correct style for if / else